EpsilonCCS/Inc/TelemetryReporting/TelemetryUtils.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#define CHECKSUM_LENGTH (2)
+#define FRAMING_LENGTH_INCREASE (2)
+#define TERMINATING_BYTE (0x00)
+
+union DataUnion
```

Comment:
This union could be moved to the .c file, other files don't need to reference it. Although it is fine here as well if you wish to keep all types defined within a header file.